.gitignore

```diff
@@ -137,3 +137,13 @@ nohup.out
 # AcadeMeData app
 AcadeMeData/.idea/
 
+# AcadeMe app - all of the .idea files that created by pycharm
+AcadeMe/.idea/

```

Comment:
As per this 
https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring

the following should do the trick for all subdirectories, I think
**/.idea

Although this template show it explicitly
https://github.com/github/gitignore/blob/master/Android.gitignore#L41_L52

Reply:
.idea/* should do the trick